exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "TestSingleBookPrice",
+            "expected": 8.0
+         },
+         {
+            "description": "TestBasketContainsMyBooks",
```

Comment:
previous discussion: https://github.com/exercism/x-common/pull/465#discussion_r91590666

I was confused about this one since the key is "mybasket" instead of "basket". Is this a significant difference?

In addition, I don't think it is particularly necessary for this problem that a basket can report what it contains - the test code is already telling the basket what it contains. It seems the tests should be interested in finding out the price of a basket (as all the other tests are).